k8s/observables_deployment.yaml

```diff
@@ -42,5 +42,12 @@ spec:
                 secretKeyRef:
                   name: elasticsearch-password
                   key: password
+          resources:
+            limits:
+              cpu: '2'
+              memory: 6Gi
+            requests:
+              cpu: 500m
+              memory: 2Gi
```

Comment:
why are these values different from the dashboard? https://github.com/greymatter-io/gitops-core/blob/main/k8s/outputs/dashboard.cue#L53-L54